packages/eslint-plugin/docs/rules/class-methods-use-this.md

```diff
@@ -41,10 +41,12 @@ class X {
 
 ### `ignoreClassesThatImplementAnInterface`
 
-Makes the rule ignore all class members that are defined within a class that `implements` a type.
+Makes the rule ignore class members that are defined within a class that `implements` a type.
```

Comment:
[Docs] The headings here are informative, but kind of spaced out I think it'd be good to have it explicitly say what they are. What do you think about this?

```suggestion
Makes the rule ignore class members that are defined within a class that `implements` a type.
If specified, it can be either:

* `true`: Ignore all classes that implement an interface
* `'public-fields'`: Ignore only the public fields of classes that implement an interface
```

Reply:
addressed in https://github.com/typescript-eslint/typescript-eslint/pull/7705/commits/316b1707007959efd3cd5c728fd6f71fc8118eb7

Reply:
Thanks!